### ISSUE TITLE
ImportError: No module named win32api

### ISSUE THREAD

**Issue by himanshu9345** [gorchard/event-Python#14]
did you use windows for developing and testing this library?
I am integrating this package in my one of my project but I am facing an issue with that.
**win32api** is library only for windows I think it's not supported in Ubuntu, can you guide me how did you test this project?

**Comment by gorchard** [comment]
Correct, this is for automatically detecting the resolution of the screen in lines 265-266. Which is useful for calibration.
    screen_width_pixels = GetSystemMetrics(0)
    screen_height_pixels = GetSystemMetrics(1)
If you manually write the screen size, you should be able to comment out the import line without any trouble.

I should also note this is very old code that isn't really used or supported anymore. Even my own lab now uses the event-based vision ROS library. Take a look at the "Software" section at https://github.com/uzh-rpg/event-based_vision_resources for some useful tools.
